gradle-jdks-integration-tests/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'java'
+apply plugin: 'org.unbroken-dome.test-sets'
+
+testSets {
+    jdkIntegrationTest
+}
+
+dependencies {
+    jdkIntegrationTestImplementation project(":gradle-jdks")
+    jdkIntegrationTestImplementation project(":gradle-jdks-setup")
+
+    jdkIntegrationTestImplementation 'commons-io:commons-io'
+    jdkIntegrationTestImplementation 'com.google.guava:guava'
+    jdkIntegrationTestImplementation 'org.junit.jupiter:junit-jupiter'
+    jdkIntegrationTestImplementation 'org.assertj:assertj-core'
+}
+
+tasks.build.dependsOn tasks.jdkIntegrationTest
+tasks.jdkIntegrationTest {
+    environment("PROJECT_VERSION", project.version)
+}
+
+tasks.withType(JavaCompile) {
+    options.errorprone.disable 'StringSplitter'
```

Comment:
I think it's better to just `@SuppressWarnings("StringSplitter")` where you need to - or just use Guava's `Splitter` as it's usually less confusing.